src/components/tests/mocks/modifiedCharacters.ts

```diff
@@ -0,0 +1,89 @@
+import { ModifiedCharacter } from '../../../api/starWars-types'
+export const modifiedCharacters: ModifiedCharacter[] = [
+  {
+    id: 1,
+    name: 'Luke Skywalker',
+    image: 'https://vignette.wikia.nocookie.net/starwars/images/2/20/LukeTLJ.jpg',
+    gender: 'male',
+    species: 'human',
+    bmi: 24.68,
+    height: 1.72,
+    mass: 73,
+    homeworld: 'tatooine',
+    born: '-19 BBY',
+    died: '34 ABY',
+  },
+  {
+    id: 4,
+    name: 'Darth Vader',
+    image: 'https://vignette.wikia.nocookie.net/fr.starwars/images/3/32/Dark_Vador.jpg',
+    gender: 'male',
+    species: 'human',
+    bmi: 29.12,
+    height: 2.03,
+    mass: 120,
+    homeworld: 'tatooine',
+    born: '-41 BBY',
+    died: '4 ABY',
+  },
+  {
+    id: 5,
+    name: 'Leia Organa',
+    image: 'https://vignette.wikia.nocookie.net/starwars/images/f/fc/Leia_Organa_TLJ.png',
+    gender: 'female',
+    species: 'human',
+    bmi: 22.67,
+    height: 1.5,
+    mass: 51,
+    homeworld: 'alderaan',
+    born: '-19 BBY',
+  },
+  {
+    id: 6,
+    name: 'Owen Lars',
+    image: 'https://vignette.wikia.nocookie.net/starwars/images/e/eb/OwenCardTrader.png',
+    gender: 'male',
+    species: 'human',
+    bmi: 41.52,
+    height: 1.7,
+    mass: 120,
+    homeworld: 'tatooine',
+    born: '-52 BBY',
+  },
+  {
+    id: 7,
+    name: 'Beru Whitesun lars',
+    image: 'https://vignette.wikia.nocookie.net/starwars/images/c/cc/BeruCardTrader.png',
+    gender: 'female',
+    species: 'human',
+    bmi: 27.55,
+    height: 1.65,
+    mass: 75,
+    homeworld: 'tatooine',
+    born: '-47 BBY',
+  },
+  {
+    id: 9,
+    name: 'Biggs Darklighter',
+    image: 'https://vignette.wikia.nocookie.net/starwars/images/0/00/BiggsHS-ANH.png',
+    gender: 'male',
+    species: 'human',
+    bmi: 25.08,
+    height: 1.83,
+    mass: 84,
+    homeworld: 'tatooine',
+    born: '-24 BBY',
+  },
+  {
+    id: 10,
+    name: 'Obi-Wan Kenobi',
+    image: 'https://vignette.wikia.nocookie.net/starwars/images/4/4e/ObiWanHS-SWE.jpg',
+    gender: 'male',
+    species: 'human',
+    bmi: 24.45,
+    height: 1.82,
+    mass: 81,
+    homeworld: 'stewjon',
+    born: '-57 BBY',
+  },
+]
```

Comment:
[nitpick] - to tell other devs that it won't change or not meant to be mutated, you wanna declare this value with `as const` as it "locks" the value from TS perspective and make each object's values to be literal types rather than primitive ones(e.g. mass: number -> mass: 81 )

https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-inference